spec/features/current_location/user_views_nearby_alums_spec.rb

```diff
@@ -10,13 +10,22 @@
   # [] I cannot view map unless I check in (updating my location)
   # [] I should see myself on the map as well
 
-  let!(:user) { FactoryGirl.create(:user) }
-  let!(:current_location) do FactoryGirl.create(:current_location,
-    user_id: user.id)
-  end
+let!(:user) { FactoryGirl.create(:user) }
+let!(:current_location) do FactoryGirl.create(:current_location,
+  user_id: user.id)
+end
 
   scenario 'user checks in' do
-    sign_in_as(user)
+  save_and_open_page
```

Comment:
Remove debugger entry point `save_and_open_page`.<br>Use 2 (not 0) spaces for indentation.